src/esp/assets/ResourceManager.cpp

```diff
@@ -559,7 +560,10 @@ bool ResourceManager::loadRenderAsset(const AssetInfo& info) {
     AssetInfo defaultInfo(info);
     defaultInfo.overridePhongMaterial = Cr::Containers::NullOpt;
 
-    if (info.type == AssetType::FRL_PTEX_MESH) {
+    if (info.type == AssetType::PRIMITIVE) {
+      buildPrimitiveAssetData(info.filepath);
```

Comment:
Note: this assumes the `AssetAttributesManager` has already registered a primitive asset with this name. As such, this won't support RenderReplay currently.

Reply:
A possible solution to this would be the string-based construction of primitives. Since primitive attributes names encode their structure, and do not allow for renaming, we could create reverse engineer the required attributes if it is missing.

Reply:
I was thinking this might be a viable option also.

Reply:
@eundersander FYI

Reply:
Note that PR #1314 addresses this gap, thanks @jturner65!